lib/std/debug.zig

```diff
@@ -1624,9 +1649,14 @@ fn handleSegfaultWindowsExtra(info: *windows.EXCEPTION_POINTERS, comptime msg: u
         os.abort();
     } else {
         switch (msg) {
-            0 => panicExtra(null, exception_address, format.?, .{}),
-            1 => panicExtra(null, exception_address, "Segmentation fault at address 0x{x}", .{info.ExceptionRecord.ExceptionInformation[1]}),
-            2 => panicExtra(null, exception_address, "Illegal Instruction", .{}),
+            0 => panicImpl(null, exception_address, format.?),
+            1 => {
+                const format_item = "Segmentation fault at address 0x{x}";
+                var buf: [format_item.len + 64]u8 = undefined; // 64 is arbitrary, but sufficiently large
+                const to_print = std.fmt.bufPrint(buf[0..buf.len], format_item, .{info.ExceptionRecord.ExceptionInformation[1]}) catch unreachable;
+                panicImpl(null, exception_address, to_print);
+            },
+            2 => panicImpl(null, exception_address, "Illegal Instruction"),
```

Comment:
This is gross.

Possible solutions:
 - Keep this
 - make `first_trace_addr` an argument to `builtin.panic` so we don't need to use `panicImpl` here (adds a decent amount of complexity and breaks existing code)
 - don't use `exception_address`

Reply:
What's gross? I think I'm missing something. Anyway thanks for the PR, happy to continue the discussion after merging.

Reply:
In order to pass the `exception_address` we have to bypass `builtin.panic` which is potentially user provided and use the default panic implementation instead (because `builtin.panic` doesn't take a `first_trace_addr`).
This is always platform supported because this function is windows only.
So, if the end user wanted special panic behavior on windows, it wouldn't also get triggered on segfaults.

Reply:
I see, thanks for the explanation. We'll definitely want to do a design pass over the panic infrastructure before stabilizing std.